src/helm/env.d/dev/values.desk.yaml.gotmpl

```
@@ -47,7 +47,10 @@ backend:
         name: backend
         key: OIDC_RP_CLIENT_SECRET
     OIDC_RP_SIGN_ALGO: RS256
-    OIDC_RP_SCOPES: "openid email siret"
+    OIDC_RP_SCOPES: "openid email siret given_name usual_name"
```

Scopes don't work like that
```suggestion
    OIDC_RP_SCOPES: "openid email siret profile"
```